lib/player.rb

```diff
@@ -0,0 +1,8 @@
+class Player
+  attr_accessor :name, :letter, :array
```

Comment:
Kindly try and use a class without attr_accessor this is because you allow your variables to be over-written from anywhere in the application
![attr_accessor](https://user-images.githubusercontent.com/32361963/96964803-c6467400-1513-11eb-9960-c44fe4ef9cdf.png)

Reply:
Thank you Peter for your detailed review. We made the changes and requested for another review